zx.mjs

```diff
@@ -103,3 +105,12 @@ async function writeAndImport(filepath, script) {
     await fs.rm(filepath)
   }
 }
+
+function checkNodeVersion(expectedNodeVersion) {
+  if (!semver.satisfies(nodeVersion, expectedNodeVersion, { includePrerelease: true })) {
+    console.log(chalk.red(
+      `You are using Node ${nodeVersion}, but zx expected Node ${expectedNodeVersion}.\nPlease upgrade your Node version.`
```

Comment:
I think this is not needed. Can we rely on package.json check instead?

Reply:
I've thought about it, but if we only use `engines`,  this field is advisory only and will only produce warnings. Or you would like to use `engine-strict` instead?

In my opinion, it's better to use `engines` field and we check NodeJS version(and maybe other checks in future) in runtime.It is actually a common concern, and we can check other potential version requires in future.

different thought maybe?